packages/microservices/server/server-kafka.ts

```diff
@@ -207,7 +207,7 @@ export class ServerKafka extends Server implements CustomTransportStrategy {
       },
       this.options.send || {},
     );
-    return this.producer.send(replyMessage);
+    return await this.producer.send(replyMessage);
```

Comment:
```suggestion
    return this.producer.send(replyMessage);
```

You don't need `await` here